### ISSUE TITLE
include_role : troubleshooting passing variable

### ISSUE THREAD

**Issue by laurentribot** [ansible/ansible-modules-core#5408]
##### ISSUE TYPE
- Bug Report
##### COMPONENT NAME

include_role
##### ANSIBLE VERSION

```
ansible 2.2.0.0
  config file = /home/userdev/Documents/dlp-ansible/deploy_aqz/ansible.cfg
  configured module search path = ['library']
```

(branch stable-2.2)
##### CONFIGURATION

local roles and libraries location
##### OS / ENVIRONMENT

Master: Ubuntu 16.04.2
Managed: Rhel 6.6
##### SUMMARY

role vars are not visible in vars attribute of include_role
##### STEPS TO REPRODUCE

```
- hosts: all
  gather_facts: True

  tasks:
    - include_role:
        name: "role_test_a"
      vars:
        a: "test"
```

role_test_a/tasks/main.yml :

```

---
- debug: var=a
- debug: var=c
- include_role:
    name: "role_test_b"
  vars:
    b: "{{ c }}"
```

role_test_a/vars/main.yml

```

---
c: "{{ a }} dummy"
```

role_test_b/tasks/main.yml

```

---
- debug: var=b
```
##### EXPECTED RESULTS

```
PLAY [all] *********************************************************************

TASK [setup] *******************************************************************
ok: [host]

TASK [role_test_a : debug] *****************************************************
ok: [host] => {
    "a": "test"
}

TASK [role_test_a : debug] *****************************************************
ok: [host] => {
    "c": "test dummy"
}

TASK [role_test_b : debug] *****************************************************
ok: [host] => {
    "b": "test dummy"
}

PLAY RECAP *********************************************************************
host : ok=4    changed=0    unreachable=0    failed=0   
```
##### ACTUAL RESULTS

```
PLAY [all] *********************************************************************

TASK [setup] *******************************************************************
ok: [host]

TASK [role_test_a : debug] *****************************************************
ok: [host] => {
    "a": "test"
}

TASK [role_test_a : debug] *****************************************************
ok: [host] => {
    "c": "test dummy"
}

TASK [role_test_b : debug] *****************************************************
ok: [host] => {
    "b": "VARIABLE IS NOT DEFINED!"
}

PLAY RECAP *********************************************************************
host : ok=4    changed=0    unreachable=0    failed=0   
```

**Comment by ansibot** [comment]
@ansible ping, this issue is waiting for your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@ansible, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@laurentribot: Thanks for taking the time to open this issue, but it was filed under the wrong github repository. Please create a new issue at https://github.com/ansible/ansible-modules-ansible/issues
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)
